### PR TITLE
Add django-helusers integration

### PR DESCRIPTION
**Pull request by mhieta**
This PR adds django-helusers / tunnistamo integration to project.

The Admin-login procedure it currently also adjusted to work with Tunnistamo with minor manual modifications to the auto-created user object, but the actual authentication provider is disabled for now because we don't have defined the roles / groups yet in the platform level.

Refs #LIIK-89

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -42,9 +42,10 @@ line_length = 88
 ignore=D100,D104,D105,D200,D203,D400
 
 [prequ]
+annotate = yes
 requirements =
     django-environ
-	Django~=2.2.9
+	Django~=2.2.12
```
Comment (Jogha) [inline setup.cfg:48]: I feel like this django-helusers commit should be split up into multiple commits. Now there is django updates, package additions, migrations and settings all in the same commit. Would be much nicer to have atleast the django update and prequ stuff on its own.

File: city-infrastructure-platform/settings.py [hunk 2]
```diff
@@ -61,7 +61,9 @@
 
 # Application definition
 DJANGO_APPS = [
-    "django.contrib.admin",
+    "helusers",
+    "social_django",
```
Comment (Jogha) [inline city-infrastructure-platform/settings.py:65]: In the previous commit you added this, but now you are moving it. This makes the history a bit harder to read in the future, so why not do the line change in the earlier commit?

File: city-infrastructure-platform/settings.py [hunk 3]
```diff
@@ -174,10 +174,17 @@
...
+        "rest_framework.authentication.SessionAuthentication",
+    ],
+    "DEFAULT_PERMISSION_CLASSES": ["rest_framework.permissions.IsAuthenticated"],
     "DEFAULT_FILTER_BACKENDS": ["django_filters.rest_framework.DjangoFilterBackend"],
-    "PAGE_SIZE": 10,
-    "DEFAULT_AUTHENTICATION_CLASSES": ("helusers.oidc.ApiTokenAuthentication",),
+    "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
+    "PAGE_SIZE": 30,
+    "DEFAULT_RENDERER_CLASSES": ["rest_framework.renderers.JSONRenderer"],
+    "DEFAULT_SCHEMA_CLASS": "rest_framework.schemas.coreapi.AutoSchema",
```
Comment (Jogha) [inline city-infrastructure-platform/settings.py:187]: Should these be combined with earlier commits/on its own/or some combination? Also some explanations why these changes are done would not hurt, because the referenced ticket is tunnistamo-integration and now we are changing DRF default settings that are not related to authentication.

File: city-infrastructure-platform/settings.py (hunk 2 above)
Comment (mhieta) [reply city-infrastructure-platform/settings.py:65]: It was actually removed and added again in the same commit, because helusers-packages needs to be in introduced in certain order relative to other Django-packages in order to work properly. But I will fix all these helusers-imports to be in a single commit.

File: city-infrastructure-platform/settings.py (hunk 3 above)
Comment (mhieta) [reply city-infrastructure-platform/settings.py:187]: Correct, I'll make a separate commit for this.

File: setup.cfg (hunk 1 above)
Comment (mhieta) [reply setup.cfg:48]: Good point, I'll change this.
